### ISSUE TITLE
Fails on crate with `links` in Cargo.toml

### ISSUE THREAD

**Issue by dtolnay** [kbknapp/cargo-outdated#189]
#### Cargo.toml:

```toml
[package]
name = "repro"
version = "0.0.0"
edition = "2018"
links = "testtest"
```

#### build.rs:

```rust
fn main() {}
```

#### src/lib.rs:

```rust
// empty
```

---

`cargo check` and `cargo build` are all fine with this setup. `cargo outdated` fails with:

```console
error: failed to parse manifest at `/tmp/cargo-outdatedv5o4kj/Cargo.toml`

Caused by:
  package `repro v0.0.0 (/tmp/cargo-outdatedv5o4kj)` specifies that it links to `testtest` but does not have a custom build script
```

**Comment by deg4uss3r** [comment]
Hi David! 

Looks like this is because we are not checking if the `Cargo.toml` file adds a build file and thus not copying it to the temporary project space for building. Putting in a PR now that at least solves your minimal example now.

**Comment by deg4uss3r** [comment]
Fixed in #190 please let us know of any other issues :)
